src/Compilers/CSharp/Test/IOperation/IOperation/IOperationTests_IConversionExpression.cs

```diff
@@ -947,9 +947,6 @@ static void Main(string[] args)
             Children(0)
```

Comment:
I'm wondering if perhaps the line above this (the IInvalidOperation) should be an IObjectCreationExpression now. @alrz, do you know what the underlying BoundNode is?

Reply:
The code is still invalid even though now we expect either an argument list or a type, so the behavior is unchanged.